env.d/terraform.dist

```diff
@@ -1,4 +1,5 @@
-# OVH Openstack environment
+
+# Openstack environment (used to store the state)
 OS_PROJECT_ID=
```

Comment:
No more openstack?

Reply:
We'll continue to use openstack to store the state.